### ISSUE TITLE
Maximise utility of our available domains

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#967]
swapmyvote.uk - current

swapmyvote.com - would this bring in more people, or lose SEO credit built up?
swapmyvote.co.uk - ditto
swapmyvote.org -  more accurate than com or co.uk. Good if we wanted to internationalise

swap.vote - also quite international, maybe useful for url shortening or (if we were an app) deep linking?

**Comment by tdg** [comment]
They all forward to the site except .com

**Comment by tdg** [comment]
Now fixed

**Comment by baob** [comment]
Is there anything else to do here, or can this be closed ?

**Comment by tdg** [comment]
Yes, if we decide to take some of this on, it will be what is known, I believe, in the parlance, as an epic

**Comment by aspiers** [comment]
This was in the wrong milestone.  I agree it's more like an epic / high-level goal and we'll need to break into specific tasks to make any progress.  I think the first one would probably be to set up uk.swapmyvote.org and uk.swap.vote.

Later we could aim for URLs like swap.vote/uk, which would be very nice.  However this approach of having the country as part of the path rather than as a sub-domain would require URL rewriting and redirection, since for many reasons (data privacy, security etc.) we'd need completely separate instances for each country.
